scripts/similarity_trainer.py

```diff
@@ -49,7 +49,3 @@ def main(args):
         )
 
     model.save()
-
-
-if __name__ == "__main__":
```

Comment:
You can keep this and make it call `main()`, like https://github.com/mozilla/bugbug/blob/master/scripts/trainer.py#L122.